### ISSUE TITLE
RP2350B hi pins and SPI/SPISlave/wire

### ISSUE THREAD

**Issue by NollKollTroll** [earlephilhower/arduino-pico#2461]
The fix for UART needs to be done on SPI as well, changing RP2350B -> PICO_RP2350B in file `SPI.cpp`.
I'm unsure if the files `SPISlave.cpp` and `wire.cpp` should also be updated?

**Comment by earlephilhower** [comment]
Yeah, that didn't get in my accidental `git master` commit. :laughing:
